dspace/config/modules/discovery.cfg

```diff
@@ -14,11 +14,12 @@ discovery.search.server = ${solr.server}/search
 #Char used to ensure that the sidebar facets are case insensitive
 #discovery.solr.facets.split.char=\n|||\n
 
-# index.ignore-variants = false
-# index.ignore-authority = false
+# discovery.index.ignore-variants = false
+# discovery.index.ignore-authority = false
 discovery.index.projection=dc.title,dc.contributor.*,dc.date.issued
 
-# ONLY-FOR-JSPUI: 
-# 1) you need to set the DiscoverySearchRequestProcessor in the dspace.cfg 
-# 2) to show facet on Site/Community/etc. you need to add a Site/Community/Collection
-#	 Processors plugin in the dspace.cfg
+discovery.facet.namedtype.item = 000item\n|||\nArchived###item
+discovery.facet.namedtype.workspace = 001workspace\n|||\nWorkspace###workspace
+discovery.facet.namedtype.workflow.item = 002workflow\n|||\nWorkflow###workflow
+discovery.facet.namedtype.workflow.claimed = 003workflow\n|||\nValidation###validation
+discovery.facet.namedtype.workflow.pooled = 004workflow\n|||\nWaiting for Controller###waitingforcontroller
```

Comment:
These configurations really need documentation or comments to describe them.  The `###` portion looks especially "odd" as usually a `#` is a comment, but here it seems to be part of the configuration.  In any case, comments & docs needed.

Reply:
done